### DISCUSSION TITLE
"Group By" option missing when choosing Time as x-axis in GitHub Project Insights

### DISCUSSION THREAD

**Discussion by yanghua-ola** [community/community#73129] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

In the past, I was able to choose Time as x-axis and also set a "Group By" option to get a nice stacked time-series. Somehow the option is now gone. It is still available if I choose a custom field as x-axis 

![image](https://github.com/community/community/assets/86941588/8ba1de22-a1e3-425d-b079-bffd1c482d66)
![image](https://github.com/community/community/assets/86941588/f51aad75-e88b-41bf-845b-2aa5624e6b5b)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by slinna** [comment]
Hi @yanghua-ola if an add-on is an option, I suggest taking a look at the [Analytics & Reports](https://github.com/marketplace/analytics-reports) by Screenful. It provides advanced charting for GitHub data (repos and Projects), including a stacked bar. Here's an example of a stacked bar chart grouped by time: 

![CleanShot 2023-11-07 at 10 48 48](https://github.com/community/community/assets/3374155/8fc40160-61b2-46ff-9cbf-377679f3f560)

You can [learn more about this add-on](https://screenful.com/github) and sign up for a free trial to give it a try.

**Comment by efasel** [comment]
We are having the same issue. Any update on this?

**Comment by joepavitt** [comment]
Worth pointing out that the [GH Insights documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/viewing-insights-from-your-project/about-insights-for-projects#:~:text=To%20create%20a%20historical%20chart%2C%20set%20your%20chart%27s%20X%2Daxis%20to%20%22Time.%22) still explicitly point this out as an option, but it's not showing at all for me on a new insight.

**Comment by KjetilIN** [comment]
Also having the same issue. Trying to plot a graph over time, but the time option is not available over the x-axis: 
![image](https://github.com/community/community/assets/66110094/53d34f42-9d49-4dee-9890-52cab249613f)

Do you need a subscription to github to get this to work?

**Comment by dehypnosis** [comment]
I purchased paid plan for this feature but it is not working.
Hey team! plz would you give any feedback?

**Comment by nickleach** [comment]
Any update on this? Not seeing this either

**Comment by MariadalitAtWork** [comment]
Same here, any update?
